indexing-handlers/src/main/java/no/unit/nva/indexing/utils/ResourceExpansion.java

```diff
@@ -0,0 +1,103 @@
+package no.unit.nva.indexing.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.IntNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.util.Spliterators;
+import java.util.stream.StreamSupport;
+
+public class ResourceExpansion {
```

Comment:
Here we affect a representation of Publication, which is the responsibility of publication-api. Adding this to search isn't acceptable.